README.md

```diff
@@ -18,8 +18,11 @@ pip install amdirt
 
 ### 2. With conda
 
+Installing AMDirT in a dedicated conda environment
```

Comment:
```suggestion
Installing AMDirT in a dedicated [conda](https://docs.conda.io/projects/miniconda/en/latest/index.html) environment
```

Please add a a corresponding link for pip installation

Reply:
Done